superset-frontend/src/components/Datasource/DatasourceEditor.jsx

```diff
@@ -1003,6 +1004,16 @@ class DatasourceEditor extends React.PureComponent {
           )}
           control={<CheckboxControl controlId="normalize_columns" />}
         />
+        <Field
+          inline
+          fieldKey="time_secondary_column"
+          label={t('Time secondary column')}
+          description={t(
+            `When this attribute is true, wheneve the secondary dttm columns are filtered,
```

Comment:
```suggestion
            `When this attribute is true, whenever the secondary dttm columns are filtered,
```

Reply:
I would maybe even recommend rephrasing, to fit the syntax of the other checkboxes, something like:
`When the secondary datetime columns are filtered, apply the same filter to the main datetime column.`